t/mojo/morbo.t

```diff
@@ -117,6 +117,7 @@ EOF
   ok((stat $script)[9] == $mtime, 'modify time has not changed');
   isnt((stat $script)[7], $size, 'size has changed');
   sleep 0.1 until -f $started;
+  $started = $started->sibling('started4.txt');
```

Comment:
There is no reason for this to be so far from the place where it is actually used.